src/helm/common/optional_dependencies.py

```diff
@@ -0,0 +1,10 @@
+class OptionalDependencyNotInstalled(Exception):
+    pass
+
+
+def handle_module_not_found_error(e: ModuleNotFoundError):
+    # TODO: Ask user to install more specific optional dependencies
+    # e.g. crfm-helm[plots] or crfm-helm[server]
+    raise OptionalDependencyNotInstalled(
+        f"Optional dependency {e.name} is not installed. " "Please run `pip install helm-crfm[all]` to install it."
```

Comment:
We could ask them to look at `setup.cfg` to determine which optional dependencies to install rather than always defaulting to `[all]`.